source/adapter.js

```diff
@@ -1069,22 +1069,18 @@ if (navigator.mozGetUserMedia) {
         element.setStreamId(streamId);
       }
       var newElement = document.getElementById(elementId);
-      newElement.onplaying = (element.onplaying) ? element.onplaying : function (arg) {};
-      newElement.onplay    = (element.onplay)    ? element.onplay    : function (arg) {};
-      newElement.onloadedmetadata    = (element.onloadedmetadata)    ? element.onloadedmetadata    : function (arg) {};
-      newElement.onclick   = (element.onclick)   ? element.onclick   : function (arg) {};
-      if (isIE) { // on IE the event needs to be plugged manually
-        newElement.attachEvent('onplaying', newElement.onplaying);
-        newElement.attachEvent('onplay', newElement.onplay);
-        newElement.attachEvent('onloadedmetadata', newElement.onloadedmetadata);
-        newElement._TemOnClick = function (id) {
-          var arg = {
-            srcElement : document.getElementById(id)
-          };
-          newElement.onclick(arg);
-        };
-      }
-      
+      var elemBuff = element;
+
+      do {
```

Comment:
while(var elemBuff = Object.getPrototypeOf( elemBuff ))  works.
I can't find a way to use for loop : we need to access the prototype of the prototype of the prototype... of the object. I don't think it's doable with a for loop in a clean way.